localization:

  # Compute ICP covariance and condition number
  compute_icp_covariance: false

  # Compute ICP observability 
  compute_icp_observability: true

  # Stop ICP if the transformation from the last iteration was this small.
  tf_epsilon: 0.0000000001

  # During ICP, two points won't be considered a correspondence if they are
  # at least this far from one another.
  corr_dist: 0.2

  # Iterate ICP this many times.
  iterations: 20 # 20 # Default from original branch is 10

  # Maximum acceptable incremental rotation and translation.
  transform_thresholding: false
  max_translation: 0.1 # 0.05
  max_rotation: 0.1

  # Number of threads GICP can use (3 threads ~ 3x speedup)
  # num_threads: 2 - defined in launch files to adapt to robot

  # Enable GICP timing output
  enable_timing_output: false
  
  # Radius for normal search
  normal_search_radius: 10
